Gemfile

```diff
@@ -74,7 +74,6 @@ end
 
 group :test do
   gem 'capybara'
-  gem 'capybara-screenshot'
```

Comment:
Screenshots of failing specs are automatically set up by Rails' built-in system test helpers.